### PR TITLE
Fix LexicalRichText.d.ts any type

### PR DESCRIPTION
**Pull request by zurfyx**
Import missing

Closes https://github.com/facebook/lexical/issues/1821

### CODE REVIEW

File: packages/lexical-rich-text/LexicalRichText.d.ts [hunk 1]
```diff
@@ -12,7 +12,10 @@ import type {
   LexicalNode,
   NodeKey,
   ParagraphNode,
+  HeadingNode,
```
Comment (Proful) [inline packages/lexical-rich-text/LexicalRichText.d.ts:15]: Facing same issue with QuoteNode and CodeNode

Comment (zurfyx) [reply packages/lexical-rich-text/LexicalRichText.d.ts:15]: Thanks!

Comment (zurfyx) [reply packages/lexical-rich-text/LexicalRichText.d.ts:15]: We addressed Code in https://github.com/facebook/lexical/pull/1818, should be available in the next release that should happen within 2 days

Comment (Proful) [reply packages/lexical-rich-text/LexicalRichText.d.ts:15]: Thanks

**Review by acywatson — APPROVED** [review]

**Pull request merged**